lib/propolis/Cargo.toml

```diff
@@ -61,3 +61,5 @@ falcon = ["libloading", "p9ds", "dlpi", "ispf", "rand", "softnpu", "viona_api/fa
 # TODO until crucible#1280 is addressed, enabling Nexus notifications is done
 # through a feature flag.
 omicron-build = ["crucible/notify-nexus"]
+
+failure-injection = []
```

Comment:
Add a little comment here about the purpose of the feature?

Reply:
I'll do one better - I thought there was `omicron-build` code in `propolis` that would move to `failure-injection`, but it turns out that there isn't (yet?)

so I was premature in adding this flag in both crates, and I'll back this one out.

Reply:
https://github.com/oxidecomputer/propolis/pull/878/commits/cf02de70e02d59ecb8fbf48fc8537c42fd6a6755 - comment on `bin/propolis-server/Cargo.toml` seems good though right?

Reply:
Yes, it's appreciated.